core/templates/domain/topic/TopicObjectFactory.ts

```diff
@@ -69,21 +72,21 @@ export class Topic {
   _nextSubtopicId: number;
   _version: number;
   _subtopics: Subtopic[];
-  _thumbnailFilename: string;
+  _thumbnailFilename: string | null;
```

Comment:
Why can this be `null`?

Reply:
Done add comment of usage of `null`.
same, is it okay if i add TODO comment for refactor the code so that the initialization with null is not needed.

Reply:
Sure.

Reply:
Done